charts/flipt/templates/service.yaml

```
@@ -1,10 +1,33 @@
+{{- if .Values.service.enabled }}
```

As this (`service.enabled`) is a new property, could we perhaps make it so that it renders the service if this value is not defined. For  backwards compatibility.

As in:

```suggestion
{{- if (or (empty .Values.service.enabled) .Values.service.enabled))}}
```